src/content/components/TimelineViewport.js

```diff
@@ -17,7 +17,9 @@ import type {
 import type { UpdateProfileSelection } from '../actions/profile-view';
 import type { ProfileSelection } from '../actions/types';
 
-const { DOM_DELTA_PAGE, DOM_DELTA_LINE } = new WheelEvent('mouse');
+const { DOM_DELTA_PAGE, DOM_DELTA_LINE } = (typeof window === 'object' && window.WheelEvent)
```

Comment:
or you can add the WheelEvent component in `window` ? (or file a bug at https://github.com/tmpvar/jsdom ? or even better, do a PR there ;) )

Please at least file a bug there